privacyidea/static/components/config/states/states.js

```diff
@@ -132,7 +132,7 @@ angular.module('privacyideaApp.configStates', ['ui.router']).config(
                     templateUrl: configpath + "config.system.html"
                 })
                 .state('config.policies', {
-                    url: "/policies",
+                    url: "/policies/",
```

Comment:
Hm, on my local development server, I end up with a weird URL for the policies list: http://localhost:5000/#/config/policies//list (with a double slash). Maybe this is related to the change above?

Reply:
I most probably is!
Where do you get this URL? (What was your "click way")

Reply:
done in da05d0b7